app/routes/__index/dns-records/instructions.tsx

```diff
@@ -1,21 +1,153 @@
+            </ListItem>
+            <ListItem>
+              Hit the "Create your first DNS Record!" button: On the DNS records page, click the
+              "Create your first DNS Record!" button to open the DNS record creation form.
+            </ListItem>
+            <ListItem>
+              Put appropriate values in fields: Fill in the required information for the selected
+              record type. (Refer to the previous section to learn about DNS record types and their
+              corresponding values.)
+            </ListItem>
+            <ListItem>
+              Hit the "Create" button: After entering the required information, click the "Create"
+              button to generate your DNS record.
+            </ListItem>
+          </OrderedList>
+          <Text>
+            Remember to test your project's web address to ensure it is accessible and functioning
+            correctly. Note that it may take some time for DNS changes to propagate.
+          </Text>
+          <Text>
+            For more information on DNS Records refer{' '}
+            <ChakraLink
+              href="https://www.cloudflare.com/learning/dns/dns-records"
+              isExternal
+              textDecoration="underline"
+            >
+              here.
+            </ChakraLink>
+          </Text>
+        </VStack>
+        <VStack alignItems="flex-start">
+          <Heading as="h3" size="md">
+            FAQ
+          </Heading>
+          <Accordion allowMultiple width="full">
+            <FaqAccordion title="How long does it take for DNS changes to propagate and become accessible?">
+              <Text>
+                DNS propagation time can vary depending on factors such as the specific DNS servers
+                involved, and the geographical location of users accessing the records. Typically,
+                DNS changes can take anywhere from a few minutes to 48 hours to propagate fully
+                across the internet.
+              </Text>
+              <Text mt="4">
+                To help you monitor the propagation status of your DNS records, we provide an
+                information icon <InfoOutlineIcon color="black" boxSize="4" /> next to each record.
+                Clicking on this icon will take you to a DNS record propagation map, where you can
+                track the progress of your DNS changes in real-time. This map shows you which DNS
+                servers around the world have updated their cache with your new record information,
+                allowing you to gauge when your changes will become accessible globally.
+              </Text>
+            </FaqAccordion>
+            <FaqAccordion title="Are there any limitations or restrictions on the number of DNS records I can create?">
+              Yes, there is a limitation on the number of DNS records you can create. Each user is
+              allowed to create up to 20 DNS records for their projects. This limit helps manage
```

Comment:
Don't list an actual number, since it could be different in the future.  Indicate that we show you how many more records you can create on the DNS Records table.